frontend/src/ts/commandline/index.ts

```diff
@@ -382,21 +382,11 @@ function restoreOldCommandLine(sshow = true): void {
   if (sshow) show();
 }
 
-$("#commandLine input").keyup((e) => {
+$("#commandLine input").on("input", () => {
```

Comment:
Replace this with:
```suggestion
$("#commandLine input").on("input", (e) => {
```
 and keep the other lines in.